app/src/main/java/org/kiwix/kiwixmobile/downloader/ChunkUtils.java

```diff
@@ -42,11 +44,11 @@
     for (String zim : fileNames) {
       String range;
       if (currentRange + CHUNK_SIZE >= contentLength) {
-        range = String.format("%d-", currentRange);
+        range = String.format(Locale.US, "%d-", currentRange);
```

Comment:
Is there a reason this is Locale.US?

Reply:
Yes, since I think it will work on most of the Android devices

And also in https://developer.android.com/reference/java/util/Locale
> The default locale is not appropriate for machine-readable output. The best choice there is usually Locale.US – this locale is guaranteed to be available on all devices, and the fact that it has no surprising special cases and is frequently used (especially for computer-computer communication) means that it tends to be the most efficient choice too.

Reply:
Great. Thanks for clarifying that. As this is indeed going to be read by a server then this seems correct :+1: